cibuild.cmd

```diff
@@ -18,9 +18,30 @@ if /I "%1" == "/release" set BuildConfiguration=Release&&shift&& goto :ParseArgu
 if /I "%1" == "/test32" set Test64=false&&shift&& goto :ParseArguments
 if /I "%1" == "/test64" set Test64=true&&shift&& goto :ParseArguments
 if /I "%1" == "/testDeterminism" set TestDeterminism=true&&shift&& goto :ParseArguments
+
+REM /buildTimeLimit is the time limit, measured in minutes, for the Jenkins job that runs
+REM the build. The Jenkins script netci.groovy passes the time limit to this script.
+if /I "%1" == "/buildTimeLimit" set BuildTimeLimit=%2&&shift&&shift&&goto :ParseArguments
+
```

Comment:
The section above needs to be submitted first as a separate change to the stabilization branch.  Otherwise the groovy change will break stabilization jobs by virtue of adding a new parameter.